Resources/Prototypes/Body/Parts/animal.yml

```
@@ -45,6 +45,7 @@
   - type: BodyPart
     partType: Hand
     symmetry: Left
+    slotId: hands # Shitmed
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)